README.md

```diff
@@ -20,95 +20,63 @@ More information about releases in Octopus Deploy:
-| `timeout`                      | A timeout value for network operations (in seconds).                                                                                                                                                                                                                 |   `600`    |
-| `username`                     | The username used to authenticate with Octopus Deploy. You must provide `api_key` or `username` and `password`. It is strongly recommended to retrieve this value from a GitHub secret.                                                                              |            |
-| `variable`                     | **❗️ Deprecated; please use variables instead.** Values for any prompted variables (format: `Label:Value`). For JSON values, embedded quotation marks should be escaped with a backslash.                                                                           |            |
-| `variables`                    | Multi-line list of values for any prompted variables (format: `Label:Value`). For JSON values, embedded quotation marks should be escaped with a backslash. Supports multiple values, one per line.                                                                  |            |
-| `wait_for_deployment`          | Wait for deployment to finish.                                                                                                                                                                                                                                       |  `false`   |
-| `what_if`                      | Perform a dry run; do not create or deploy a release.                                                                                                                                                                                                                |  `false`   |
+| Name | Description |
+| :--- | :---------- |
+| `project` | **Required.** The name or ID of the project associated with this release. |
+| `api_key` | The API key used to access Octopus Deploy. An API key is required, but you may also use the `OCTOPUS_API_KEY` environment variable. If the guest account is enabled, a key of API-GUEST may be used. It is strongly recommended that this value retrieved from a GitHub secret. |
+| `server` | The base URL hosting Octopus Deploy (i.e. "https://octopus.example.com/"). The Server URL is required, but you may also use the `OCTOPUS_HOST` environment variable. |
+| `release_number` | The number for the new release. If omitted, Octopus Deploy will generate a release number. |
+| `space` | The name or ID of a space within which this command will be executed. If omitted, the default space will be used. |
```

Comment:
Let's remove the text, "If omitted, the default space will be used."

Also, we'll need to cite the environment variable, `OCTOPUS_SPACE` like we do in the description for `api_key` (above).